src/utils/constants.js

```diff
@@ -40,61 +40,62 @@ export const WEBCLIENT = "WEBCLIENT";
  * @type {Object}
  */
 export const REQUEST_PARAMS = {
-    DATASET_ID : 'DATASET',
-    SERVER : 'SERVER',
-    IMAGE_ID : 'IMAGE_ID',
-    CHANNELS : 'C',
-    PLANE : 'Z',
-    TIME : 'T',
-    PROJECTION : 'P',
-    MODEL : 'M',
-    CENTER_X : 'X',
-    CENTER_Y : 'Y',
-    ZOOM : 'ZM'
+    CHANNELS: 'C',
```

Comment:
@jburel one is the internal ol3-viewer which is compiled by closure and can be run by itself. so they also have different namespacing, ecma script standards and even code. I'm not super happy about the duplication in some places but it's the nature of the beast.